FfiSpike/FfiSpike.cpp

```diff
@@ -548,21 +550,52 @@ HANDLE start_debug_server(std::string device_identifier, std::string ddi, std::s
 	return gdb;
 }
 
-afc_connection *get_afc_connection(std::string device_identifier, const char* application_identifier, std::string method_id)
+bool start_afc_client(std::string& device_identifier, std::string& root_path, const char* application_identifier, HANDLE house_arrest_fd, std::string& method_id)
```

Comment:
I'd suggest renaming `root_path` to `destination` as it is semantically more correct.